### ISSUE TITLE
[Telemetry]  In progress  media tile

### ISSUE THREAD

**Issue by Mugurell** [mozilla-mobile/fenix#20393]
## Description & Product Manager / Data Scientist User Story
As a Product Manager I want to collect telemetry data on how often users tap on the in progress media tile on the homescreen

## What questions will you answer with this data?
- How useful is the in progress media tile

## Acceptance Criteria
- [ ] ENG files a [DS JIRA](https://jira.mozilla.com/projects/DO/issues/DO-228?filter=allopenissues) request outlining their methodology.
- [ ] DS sign off on instrumentation methodology addressing product questions.
- [ ] Event pings can be queried via re:dash
- [ ] Event pings can be queried via amplitude
- [ ] We are sending telemetry events for the actions listed in the requirements
- [ ] We have documented the telemetry
- [ ] We have asked a data steward to [review](https://github.com/mozilla/data-review/blob/master/request.md) the telemetry

Tracking https://mozilla-hub.atlassian.net/browse/FNXV2-17118

**Comment by Mugurell** [comment]
This is still waiting on https://github.com/mozilla-mobile/fenix/pull/20138 and it would probably be best if both tickets would be resolved in the same PR.

**Comment by lobontiumira** [comment]
Verified as fixed on the 8/2 Nightly build 92.0a1 with GV 92.0a1, with Google Pixel (Android 10).
Ping: 9abd6ff4-45ab-488a-bc28-cb067370ecf9

  ```
{
          "category": "recent_tabs",
          "name": "in_progress_media_tab_opened",
          "timestamp": 11553
        }
```

[Glean dashboard](https://debug-ping-preview.firebaseapp.com/pings/mira-2108020001)
